specification/http/src/main/scripts/org/kaazing/specification/http/rfc7232/headers/get.request.with.if.modified.since.header/response.rpt

```diff
@@ -0,0 +1,22 @@
+accept http://localhost:8000/resource
+accepted
+connected
+
+read method "GET"
+read version "HTTP/1.1"
+
+write status "200" "OK"
+write header "Last-Modified" "Mon, 1 Jan 2015 01:23:45 GMT"
```

Comment:
create a property called lastModified and write that property and use the same property for reading as well